lib/ae_mdw_web/websocket/broadcaster.ex

```diff
@@ -232,4 +249,18 @@ defmodule AeMdwWeb.Websocket.Broadcaster do
     |> Enum.map(&AeMdw.Validate.id!/1)
     |> Enum.uniq()
   end
+
+  defp already_processed?(%__MODULE__{dispatched_hashes: dispatched_hashes}, type_hash) do
+    :queue.member(type_hash, dispatched_hashes)
+  end
+
+  defp push_hash(%__MODULE__{dispatched_hashes: dispatched_hashes}, type_hash) do
+    queue = :queue.in(type_hash, dispatched_hashes)
+
+    if length(:queue.to_list(queue)) > @max_hashes_size do
```

Comment:
Could you increase this number please? I counting I am doing is 10 generations (max of sync `Server`), 20 microblocks and more than 50 transactions each.

Reply:
Or maybe use some kind of TTL to remove the older ones.

Reply:
Moved to ETS for the TTL expiring. thanks!